src/app/hero-detail/hero-detail.component.html

```diff
@@ -1 +1,9 @@
-<p>hero-detail works!</p>
+<div *ngIf="hero">
+
+  <h2>{{hero.name | uppercase}} Details</h2>
```

Comment:
It doesn't look like you've got prettier set up in this repo? These aren't formatted to it's standard.